templates/components/category/sort-box.html

```diff
@@ -1,36 +1,36 @@
-<fieldset class="js-form-fieldset">
-    <label for="sort">
-        {{lang 'common.sorter.sort_by'}}
-    </label>
-    <select name="sort" id="sort" role="listbox">
-        <option value="featured"{{#if sort '==' 'featured'}} selected{{/if}}>
-            {{lang 'common.sorter.featured'}}
-        </option>
-        <option value="newest"{{#if sort '==' 'newest'}} selected{{/if}}>
-            {{lang 'common.sorter.newest'}}
-        </option>
-        <option value="bestselling"{{#if sort '==' 'bestselling'}} selected{{/if}}>
-            {{lang 'common.sorter.top_sellers'}}
-        </option>
-        <option value="alphaasc"{{#if sort '==' 'alphaasc'}} selected{{/if}}>
-            {{lang 'common.sorter.alpha_asc'}}
-        </option>
-        <option value="alphadesc"{{#if sort '==' 'alphadesc'}} selected{{/if}}>
-            {{lang 'common.sorter.alpha_desc'}}
-        </option>
-        <option value="avgcustomerreview"{{#if sort '==' 'avgcustomerreview'}} selected{{/if}}>
-            {{lang 'common.sorter.reviews'}}
-        </option>
-        <option value="priceasc"{{#if sort '==' 'priceasc'}} selected{{/if}}>
-            {{lang 'common.sorter.price_asc'}}
-        </option>
-        <option value="pricedesc"{{#if sort '==' 'pricedesc'}} selected{{/if}}>
-            {{lang 'common.sorter.price_desc'}}
-        </option>
-        {{#if result_count}}
-            <option value="relevance"{{#if sort '==' 'relevance'}} selected{{/if}}>
-                {{lang 'common.sorter.relevance'}}
-            </option>
-        {{/if}}
-    </select>
-</fieldset>
+<fieldset class="js-form-fieldset">
+    <label for="sort c-form__label">
```

Comment:
@mgbelegu please move the selector to the class attribute ;).

Reply:
@marcinkrzeminski my bad 😅 great catch though